packages/react/src/icon/docs/overview.mdx

```diff
@@ -17,30 +17,33 @@ relatedComponents:
+Meaningful icons can be used to enhance the experience for sighted users; making it easier to scan content, identify actions and understand user interfaces. The icon component gives you access to a suite of system icons that are designed to represent common states and universal actions.
 
 <DoHeading />
 
-- Only use universal icons, that all users understand, and that clearly represent an intended meaning or context
+- Use icons, that all users understand, and that clearly represent an intended meaning or context.
 - Use icons to support visual recognition of repeatable or consistent system wide states or actions
 - Always use a meaningful text label and accessible name when using an icon in an action
 - Avoid using bespoke icons in your applications as you may confuse users
-- Only consider using an icon without a meaningful label if there is insufficient space and including the label will detract from the user experience.
-  - An example of this is mobile search bar where including a label for the action will take up essential space for search input, and the icon meaning is explicit like ’search’ or ’close’.
+- Only consider using an icon without a visible label if there is insufficient space and including the label will detract from the user experience.
 
 <DontHeading />
 
 - Use icons that are unconventional or are not commonly understood to represent the action or state - this will confuse users.
 
 ## Icons
 
-When using an icon, always provide a meaningful label and accessible name to help all users quickly recognise and understand what action will occur. Don’t use icons without labels because you may confuse some users - even the simplest icons will be new or uncommon to some users.
+Each icon in our library is simple but detailed enough to ensure users can understand them. Icons should not be too detailed, so they are clear and legible at all available sizes. [See the full list of available icons.](/foundations/icons)
 
-Each icon in our library is meaningful and universal, to ensure users understand them. Icons should not be too detailed, to ensure they’re clear and legible at all available sizes. [See the full list of available icons.](/foundations/icons)
+> Do you have a user need for a new or missing icon? [Get in touch](/about) to discuss an icon contribution.
 
-Use icons that accurately communicate the action with a label that describes the action. For example, search fields on mobile often use a magnifying glass icon without a label due to space constraints.
+## Icons and labels
 
-> Do you have a user need for a new or missing icon? [Get in touch](/about) to discuss an icon contribution.
+Provide a meaningful label and accessible name with the icon to help all users quickly recognise and understand what action will occur. Icons are small visual elements that can be interpreted differently by people. Using icons without labels can confuse some users - even the simplest icons will be new or uncommon to some users.
```

Comment:
We don't always provide an accessible because often the label is the the thing
```suggestion
Provide a meaningful label and/or accessible name with the icon to help all users quickly recognise and understand what action will occur. Icons are small visual elements that can be interpreted differently by people. Using icons without labels can confuse some users – even the simplest icons will be new or uncommon to some users.
```